tools/PI/DevHome.PI/Controls/AddToolControl.xaml.cs

```diff
@@ -84,7 +84,12 @@ private void HandleBrowseButton()
     {
         // WinUI3's OpenFileDialog does not work if we're running elevated. So we have to use the old Win32 API.
         var fileName = string.Empty;
-        var filter = "Executables (*.exe)\0*.exe\0Batch Files (*.bat)\0*.bat\0\0";
+        var filter = "Executables (*.exe)\0*.exe\0" +
```

Comment:
NIT:
1. Use @ to avoid +.  Each + equates to throwing away and making a new string.
2. Change to const to avoid re-calculating this every page load.

Reply:
I can't use @ because I need to retain all the explicit null terminators. I can however initialize this once up front as you suggest - fixed, thanks.